.github/workflows/auto-assign-project.yml

```diff
@@ -0,0 +1,31 @@
+name: Auto Assign to Project(s)
+
+on:
+  issues:
+    types: [opened, labeled]
+  pull_request:
+    types: [opened, labeled]
+  issue_comment:
+    types: [created]
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to One Project
+    steps:
+    - name: Assign NEW issues and NEW pull requests to project 2

```

Comment:
The name says 'project 2' but the url to the project says 'https://github.com/orgs/eclipse-volttron/projects/3'.

Reply:
yeah it's copy paste....waiting until the [this issue](https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1647#note_960481) is done before updating comment.